1_space_odyssey/index.html

```diff
@@ -10,7 +10,7 @@
 </head>
 <body class="fade-in">
     
-    <script src="./js/mannuel-9000/app.js"></script>
+    <script src="./js/mannuel-9000/peder-9000/app.js"></script>
```

Comment:
please put your named folder into the root of the JS folder and leave JS include in place.
so you path should be 
`js/peder-9000/app.js`